packages/config/config/devices/0x0090/templates/kwikset_template.json

```diff
@@ -0,0 +1,887 @@
+// Kwikset Template File
+{
+	"usercode_1": {
+		"label": "User Code 1 Type",
+		"description": "Guests and Workers will require schedules attached to them in order for those types to be assigned",
+		"valueSize": 1,
+		"minValue": 1,
+		"maxValue": 4,
+		"defaultValue": 1,
+		"unsigned": true,
+		"readOnly": false,
+		"writeOnly": false,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Owner",
+				"value": 1
+			},
+			{
+				"label": "Guest",
+				"value": 3
+			},
+			{
+				"label": "Worker",
+				"value": 4
+			}
+		]
+	},
```

Comment:
I'd define this once and then import it from the `usercode_x` keys, overwriting the label.